src/wp-content/themes/twentytwenty/style.css

```
@@ -3075,6 +3075,14 @@ ol.wp-block-latest-comments {
 	font-size: inherit;
 }
 
+:root :where(.wp-block-calendar[style*="font-weight"]) caption, 
+:root :where(.wp-block-calendar *[style*="font-weight"]) ~ caption {
+    font-weight: inherit;
+}
+
+.wp-calendar-nav .wp-calendar-nav-prev {
+	font-family: "Inter var", -apple-system, BlinkMacSystemFont, "Helvetica Neue", Helvetica, sans-serif;
+}
```

```suggestion
:where(.wp-block-calendar[style*="font-weight"]) caption {
	font-weight: inherit;
}

```

This section can be simpler:
1. The `caption` only needs the specificity of one element to override the theme's default `600` font weight.
2. The `:where(.wp-block-calendar *[style*="font-weight"]) ~ caption` selector would not match any elements because the Calendar `caption` does not have siblings before it.
3. Instead of rewriting the "Inter var" font list in this section, you could add `.entry-content .wp-block-calendar` (between archives and categories) in [**two** lists of block selectors](https://github.com/WordPress/wordpress-develop/blob/b8f28f9ade2e88163f11ee31efb39a7ff04a4f48/src/wp-content/themes/twentytwenty/style.css#L2837-L2862) plus the [PHP array](https://github.com/WordPress/wordpress-develop/blob/b8f28f9ade2e88163f11ee31efb39a7ff04a4f48/src/wp-content/themes/twentytwenty/classes/class-twentytwenty-non-latin-languages.php#L121). Then the font would apply to both the previous and next months' links.